app/helpers/application_helper.rb

```diff
@@ -543,5 +543,29 @@ def cancel_button_component(class_name: nil, id: , value:, data: nil)
       end
     end
   end
+  def taxonomy_card(taxonomy:)
+    content_tag(:div, class: 'taxonomy-card') do
```

Comment:
for this you will need to create a view component as to complex for just a helper